WhateverGreen/kern_igfx.cpp

```diff
@@ -593,6 +605,10 @@ bool IGFX::processKext(KernelPatcher &patcher, size_t index, mach_vm_address_t a
 			}
 		}
 
+		// We could iterate through each submodule and redirect the request if and only if the submodule is enabled
+		if (modDVMTCalcFix.enabled)
+			modDVMTCalcFix.processKext(patcher, index, address, size);
```

Comment:
`processKext` is clearly wrong for the name of this function, as it only processes the framebuffer kext. At the very least I will ask it to be renamed to `processFramebufferKext`.

Reply:
`processKext()` is defined in the interface `PatchSubmodule`. From the perspective of `PatchSubmodule`, it is possible that another concrete type implements this function to process the graphics accelerator kext instead.

Or do you suggest that we should modify the `PatchSubmodule` interface to separate kext processing into two functions, `processFramebufferKext()` and `processAcceleratorKext()`?

i.e. The new interface definition is as follows.

```
class PatchSubmodule {

    // ... Omitted ...

    virtual void processKernel(KernelPatcher &patcher, DeviceInfo *info) = 0;

    virtual void processFramebufferKext(KernelPatcher &patcher, size_t index, mach_vm_address_t address, size_t size) = 0;

    virtual void processAcceleratorKext(KernelPatcher &patcher, size_t index, mach_vm_address_t address, size_t size) = 0;
};
```

Reply:
Probably the latter.

Reply:
Done.